pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -1205,8 +1213,12 @@ func (b *SlirpBindMechanism) discoverPodNetworkInterface() error {
 	return nil
 }
 
-func (b *SlirpBindMechanism) preparePodNetworkInterface() (api.Interface, error) {
-	return api.Interface{}, nil
+func (b *SlirpBindMechanism) preparePodNetworkInterface() error {
+	return nil
+}
+
+func (b *SlirpBindMechanism) generateDomainIfaceSpec() api.Interface {
+	return api.Interface{}
```

Comment:
I think we are suppose to panic if this function is called. As this is similar to the "preparePod...", it is not a must to take care of that here. A following refactor should either add a panic or just recognize this does not fit as a binding interface, just because the behavior does not exist for this specific binding.

Reply:
My goal is to remove the notion of bind mechanism; part of which is to make slirp something that is only used on what is not phase#2.

This aligns to what you're saying.